.github/workflows/build.yml

```diff
@@ -2,22 +2,56 @@ name: Build
 
 on: [ pull_request ]
 
+env:
+  NODE_ACTIVE_LTS_VERSION: 24
+  NODE_PREVIOUS_LTS_VERSION: 22
+  NPM_VERSION: 11
+
 jobs:
+  prepare_env:
+    name: Prepare environment
+    runs-on: ubuntu-24.04
+    outputs:
+      NODE_ACTIVE_LTS_VERSION: ${{ env.NODE_ACTIVE_LTS_VERSION }}
+      NODE_PREVIOUS_LTS_VERSION: ${{ env.NODE_PREVIOUS_LTS_VERSION }}
+      NPM_VERSION: ${{ env.NPM_VERSION }}
+    steps:
+      - name: Compute outputs
+        run: |
+          echo "NODE_ACTIVE_LTS_VERSION=${{ env.NODE_ACTIVE_LTS_VERSION }}" >> $GITHUB_OUTPUT
+          echo "NODE_PREVIOUS_LTS_VERSION=${{ env.NODE_PREVIOUS_LTS_VERSION }}" >> $GITHUB_OUTPUT
+          echo "NPM_VERSION=${{ env.NPM_VERSION }}" >> $GITHUB_OUTPUT
   build:
     name: Build distribution CSS and JS
     runs-on: ubuntu-24.04
+    needs: prepare_env
     strategy:
       matrix:
-        node: [ 22, 24 ]
+        node:
+          - ${{ needs.prepare_env.outputs.NODE_ACTIVE_LTS_VERSION }}
+          - ${{ needs.prepare_env.outputs.NODE_PREVIOUS_LTS_VERSION }}
     steps:
       - name: Clone repository
-        uses: actions/checkout@v4
+        uses: actions/checkout@v6
 
       - name: Set up Node.js ${{ matrix.node }}
-        uses: actions/setup-node@v4
+        uses: actions/setup-node@v6
         with:
           node-version: ${{ matrix.node }}
           cache: npm
+        if: matrix.node == env.NODE_ACTIVE_LTS_VERSION
+
+      - name: Set up Node.js ${{ matrix.node }}
+        uses: actions/setup-node@v6
+        with:
+          node-version: ${{ matrix.node }}
+          cache: npm
+        if: matrix.node == env.NODE_PREVIOUS_LTS_VERSION
+        continue-on-error: true
```

Comment:
`continue-on-error: true` is needed as Node 22 is shipped with `npm@10` and `devEngines` requires `11`. This allow us to continue to `Install npm@11 for Node.js` step where up-to-date version is installed.